src/languages/verilog.js

```diff
@@ -1,42 +1,87 @@
+      'tranif0 tranif1 tri tri0 tri1 triand trior trireg type typedef union unique unique0 ' +
+      'unsigned until until_with untyped use uwire var vectored virtual void wait wait_order ' +
+      'wand weak weak0 weak1 while wildcard wire with within wor xnor xor',
+    literal:
+      'null',
+    built_in:
+      '$finish $stop $exit $fatal $error $warning $info $realtime $time $printtimescale ' +
+      '$bitstoreal $bitstoshortreal $itor $signed $cast $bits $stime $timeformat ' +
+      '$realtobits $shortrealtobits $rtoi $unsigned $asserton $assertkill $assertpasson ' +
+      '$assertfailon $assertnonvacuouson $assertoff $assertcontrol $assertpassoff ' +
+      '$assertfailoff $assertvacuousoff $isunbounded $sampled $fell $changed $past_gclk ' +
+      '$fell_gclk $changed_gclk $rising_gclk $steady_gclk $coverage_control ' +
+      '$coverage_get $coverage_save $set_coverage_db_name $rose $stable $past ' +
+      '$rose_gclk $stable_gclk $future_gclk $falling_gclk $changing_gclk ' +
+      '$coverage_get_max $coverage_merge $get_coverage $load_coverage_db $typename ' +
+      '$unpacked_dimensions $left $low $increment $clog2 $ln $log10 $exp $sqrt $pow ' +
+      '$floor $ceil $sin $cos $tan $countbits $onehot $isunknown $fatal $warning ' +
+      '$dimensions $right $high $size $asin $acos $atan $atan2 $hypot $sinh $cosh ' +
+      '$tanh $asinh $acosh $atanh $countones $onehot0 $error $info $random ' +
+      '$dist_chi_square $dist_erlang $dist_exponential $dist_normal $dist_poisson ' +
+      '$dist_t $dist_uniform $q_initialize $q_remove $q_exam $async$and$array ' +
+      '$async$nand$array $async$or$array $async$nor$array $sync$and$array ' +
+      '$sync$nand$array $sync$or$array $sync$nor$array $q_add $q_full ' +
+      '$async$and$plane $async$nand$plane $async$or$plane $async$nor$plane ' +
+      '$sync$and$plane $sync$nand$plane $sync$or$plane $sync$nor$plane $system'
+    };
   return {
-    aliases: ['v'],
+    aliases: ['v', 'sv', 'svh'],
     case_insensitive: false,
-    keywords: {
-      keyword:
-        'always and assign begin buf bufif0 bufif1 case casex casez cmos deassign ' +
-        'default defparam disable edge else end endcase endfunction endmodule ' +
-        'endprimitive endspecify endtable endtask event for force forever fork ' +
-        'function if ifnone initial inout input join macromodule module nand ' +
-        'negedge nmos nor not notif0 notif1 or output parameter pmos posedge ' +
-        'primitive pulldown pullup rcmos release repeat rnmos rpmos rtran ' +
-        'rtranif0 rtranif1 specify specparam table task timescale tran ' +
-        'tranif0 tranif1 wait while xnor xor ' +
-        // types
-        'highz0 highz1 integer large medium pull0 pull1 real realtime reg ' +
-        'scalared signed small strong0 strong1 supply0 supply0 supply1 supply1 ' +
-        'time tri tri0 tri1 triand trior trireg vectored wand weak0 weak1 wire wor'
-    },
+    keywords: SV_KEYWORDS,
```

Comment:
The default lexing for keywords won't catch dollar signs, so please add `lexemes: /[\w\$]+/` here too.